packages/core/util/types/mst.ts

```diff
@@ -25,6 +25,8 @@ export const NoAssemblyRegion = types
     start: types.number,
     end: types.number,
     reversed: types.optional(types.boolean, false),
+    parentStart: types.optional(types.number, -1),
```

Comment:
@cmdcolin was this more what you had in mind? It can become useful in other parts instead of having to call the parentRegion helper method every time and it gets set here plugins/linear-genome-view/src/LinearGenomeView/index.ts line 512

Reply:
Would it work to call it parentRegion? I guess my thoughts were that it would be like parentRegion from calculateStaticBlocks, but then refer to the assembly level (I guess calculateStaticBlock's version of parentRegion referes to the displayedRegion that generated the block)

Reply:
what I can do is just pass the entire parent instead of individual parentStart and parentEnd?